pkg/manager/internal.go

```diff
@@ -260,6 +263,20 @@ func (cm *controllerManager) SetFields(i interface{}) error {
 	return nil
 }
 
+// AddMetricsExtraHandler adds extra handler served on path to the http server that serves metrics.
+func (cm *controllerManager) AddMetricsExtraHandler(path string, handler http.Handler) {
+	cm.mu.Lock()
+	defer cm.mu.Unlock()
+
+	_, found := cm.metricsExtraHandlers[path]
+	if found {
+		panic(fmt.Errorf("can't register extra handler by duplicate path %q on metrics http server", path))
+	}
+
+	cm.metricsExtraHandlers[path] = handler
+	log.Info("registering metrics http server extra handler", "path", path)
```

Comment:
```suggestion
	log.Info("Registering metrics http server extra handler", "path", path)
```

also maybe use V(2)?

Reply:
Done